### PR TITLE
The PodOverhead feature is GA

### PR DESCRIPTION
**Pull request by tengqm**
xref: kubernetes/kubernetes#108441

### CODE REVIEW

**Comment by jihoon-seo** [comment]
@tengqm [Netlify build log](https://app.netlify.com/sites/kubernetes-io-vnext-staging/deploys/6233d6de9cac480008b68f41#L77) says that
> "GA" is not a valid feature-state, use one of "alpha, beta, deprecated, stable"

PTAL 😊

**Comment by mehabhalodiya** [comment]
enhancement ref: https://github.com/kubernetes/enhancements/issues/688

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 1]
```diff
@@ -10,39 +10,32 @@ weight: 30
...
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.18" state="beta" >}}
-
+{{< feature-state for_k8s_version="v1.24" state="stable" >}}
 
 When you run a Pod on a Node, the Pod itself takes an amount of system resources. These
 resources are additional to the resources needed to run the container(s) inside the Pod.
-_Pod Overhead_ is a feature for accounting for the resources consumed by the Pod infrastructure
+'Pod Overhead' is a feature for accounting for the resources consumed by the Pod infrastructure
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/pod-overhead.md:17]: We should document behaviors over features.
```suggestion
In Kubernetes,  _pod overhead_ is a way to account for the resources consumed by the Pod infrastructure
```

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 2]
```diff
@@ -10,39 +10,32 @@ weight: 30
...
 [RuntimeClass](/docs/concepts/containers/runtime-class/).
 
-When Pod Overhead is enabled, the overhead is considered in addition to the sum of container
-resource requests when scheduling a Pod. Similarly, the kubelet will include the Pod overhead when sizing
-the Pod cgroup, and when carrying out Pod eviction ranking.
+A pod's overhead is considered in addition to the sum of container resource requests when
+scheduling a Pod. Similarly, the kubelet will include the Pod overhead when sizing the Pod cgroup,
+and when carrying out Pod eviction ranking.
 
 ## Enabling Pod Overhead {#set-up}
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/pod-overhead.md:31]:
```suggestion
## Configuring pod overhead {#set-up}
```
or, actually, omit the header. The text at the start of the next section makes much the same point.

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 3]
```diff
@@ -158,40 +156,43 @@ POD_ID="$(sudo crictl pods --name test-pod -q)"
...
+to help identify when Pod overhead is being utilized and to help observe stability of workloads
+running with a defined overhead. This functionality is not available in the 1.9 release of
+`kube-state-metrics`, but is expected in a following release. Users will need to build
+`kube-state-metrics` from source in the meantime.
 
 ## {{% heading "whatsnext" %}}
 
-
 * [RuntimeClass](/docs/concepts/containers/runtime-class/)
 * [PodOverhead Design](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/688-pod-overhead)
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/pod-overhead.md:197]:
```suggestion
* Read the [PodOverhead](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/688-pod-overhead) enhancement proposal for extra context
```

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 4]
```diff
@@ -158,40 +156,43 @@ POD_ID="$(sudo crictl pods --name test-pod -q)"
...
-to help identify when PodOverhead is being utilized and to help observe stability of workloads
-running with a defined Overhead. This functionality is not available in the 1.9 release of
-kube-state-metrics, but is expected in a following release. Users will need to build kube-state-metrics
-from source in the meantime.
-
-
+to help identify when Pod overhead is being utilized and to help observe stability of workloads
+running with a defined overhead. This functionality is not available in the 1.9 release of
+`kube-state-metrics`, but is expected in a following release. Users will need to build
+`kube-state-metrics` from source in the meantime.
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/pod-overhead.md:192]: > This functionality is not available in the 1.9 release of `kube-state-metrics`, but is expected in a following release. Users will need to build `kube-state-metrics` from source in the meantime.

We [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future); I'd drop this

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 5]
```diff
@@ -122,21 +115,25 @@ For CPU, if the Pod is Guaranteed or Burstable QoS, the kubelet will set `cpu.sh
...
 ```
 
 Check this against what is observed by the node:
+
 ```bash
 kubectl describe node | grep test-pod -B2
 ```
 
-The output shows 2250m CPU and 320MiB of memory are requested, which includes PodOverhead:
+The output shows 2250m CPU and 320MiB of memory are requested, which includes Pod overhead:
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/pod-overhead.md:135]: How about:
```suggestion
The output shows requests for 2.250 CPU, and for 320MiB of memory. These requests includes Pod overhead:
```

I think “2.25” is more meaningful to people who aren't familiar with Kubernetes quantities or with Docker CPU fractions.

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 6]
```diff
@@ -92,7 +85,7 @@ updates the workload's PodSpec to include the `overhead` as described in the Run
 the Pod will be rejected. In the given example, since only the RuntimeClass name is specified, the admission controller mutates the Pod
 to include an `overhead`.
 
-After the RuntimeClass admission controller, you can check the updated PodSpec:
+After the RuntimeClass admission controller, you can check the updated Pod spec:
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/pod-overhead.md:88]:
```suggestion
After the RuntimeClass admission controller has made modifications, you can check the updated
overhead value:
```

(I wonder, could we also show this with `kubectl ... --dry-run=server`?)

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 7]
```diff
@@ -10,39 +10,32 @@ weight: 30
...
-You need to make sure that the `PodOverhead`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled (it is on by default as of 1.18)
-across your cluster, and a `RuntimeClass` is utilized which defines the `overhead` field.
+You need to make sure a `RuntimeClass` is utilized which defines the `overhead` field.
 
 ## Usage example
 
-To use the PodOverhead feature, you need a RuntimeClass that defines the `overhead` field. As
-an example, you could use the following RuntimeClass definition with a virtualizing container runtime
+To use the Pod overhead feature, you need a RuntimeClass that defines the `overhead` field. As
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/pod-overhead.md:37]:
```suggestion
To work with Pod overhead, you need a RuntimeClass that defines the `overhead` field. As
```

**Review by sftim — COMMENTED** [review]
Thanks. I do have some suggestions on this one.

One extra thing I'd consider tidying up: use a made up runtime class name, not `kata-fc` (which refers to 2 specific external projects).

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md (hunk 4 above)
Comment (tengqm) [reply content/en/docs/concepts/scheduling-eviction/pod-overhead.md:192]: Double checked  the project and the metrics. The text needs some update.

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md (hunk 5 above)
Comment (tengqm) [reply content/en/docs/concepts/scheduling-eviction/pod-overhead.md:135]: I don't think so. The `2250m` expression is the new norm in Kubernetes. People have to get used to that.

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md (hunk 6 above)
Comment (tengqm) [reply content/en/docs/concepts/scheduling-eviction/pod-overhead.md:88]: I'm not sure if dry run works for this case.

File: content/en/docs/concepts/containers/runtime-class.md [hunk 8]
```diff
@@ -59,27 +59,30 @@ The RuntimeClass resource currently only has 2 significant fields: the RuntimeCl
...
-Overview](/docs/reference/access-authn-authz/authorization/) for more details.
+restricted to the cluster administrator. This is typically the default. See
+[Authorization Overview](/docs/reference/access-authn-authz/authorization/) for more details.
 {{< /note >}}
 
 ## Usage
 
-Once RuntimeClasses are configured for the cluster, using them is very simple. Specify a
-`runtimeClassName` in the Pod spec. For example:
+Once RuntimeClasses are configured for the cluster, you can pecify a
```
Comment (sftim) [inline content/en/docs/concepts/containers/runtime-class.md:84]: nit
```suggestion
Once RuntimeClasses are configured for the cluster, you can specify a
```

**Comment by nate-double-u** [comment]
Hello, this is a friendly reminder from the 1.24 docs release team that the _Docs' complete deadline_ — All PRs reviewed and ready to merge — is EOD **Tuesday, April 12th, 2022**. Please finish up any remaining technical reviews, and reach out if you need any help.

**Comment by dims** [comment]
cc @SergeyKanzhelev @ehashman

File: content/en/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 9]
```diff
@@ -10,17 +10,12 @@ weight: 30
...
 
 When you run a Pod on a Node, the Pod itself takes an amount of system resources. These
 resources are additional to the resources needed to run the container(s) inside the Pod.
-_Pod Overhead_ is a feature for accounting for the resources consumed by the Pod infrastructure
-on top of the container requests & limits.
-
-
-
-
+In Kubernetes, "Pod Overhead" is a way for to account for the resources consumed by the Pod
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/pod-overhead.md:17]: We [use italics to introduce new concepts](https://kubernetes.io/docs/contribute/style/style-guide/#use-italics-to-define-or-introduce-new-terms)
```suggestion
In Kubernetes, _Pod overhead_ is a way for to account for the resources consumed by the Pod
```

**Review by sftim — COMMENTED** [review]
/lgtm
(from a docs perspective)

One nit

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

thank you for addressing the issue with metrics rename as well

**Comment by nate-double-u** [comment]
Reapplying LGTM as the last edit was a nit fix.
/lgtm

Thanks everyone!
/approve

**Comment by tengqm** [comment]
netlify is stuck, recommitted the PR.

**Pull request merged**